src/views/CustomTileDialog/ImportExport/index.tsx

```
@@ -35,18 +41,42 @@ export default function ImportExport({
+                type: 'error',
+              });
+              return;
+            }
+            exportedData = await exportGroupData(singleGroup, settings.locale || 'en');
+            break;
+          case 'default':
+            exportedData = await exportCleanData(settings.locale || 'en', {
+              exportScope: 'default',
+            });
+            break;
+          case 'all':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)